src/Apps/__tests__/Fixtures/Auction/Routes/Register/index.ts

```diff
@@ -1,23 +1,44 @@
-import { redirects_me } from "__generated__/redirects_me.graphql"
-import { redirects_sale } from "__generated__/redirects_sale.graphql"
+import { Register_me } from "__generated__/Register_me.graphql"
+import { Register_sale } from "__generated__/Register_sale.graphql"
+import { routes_RegisterQueryResponse } from "__generated__/routes_RegisterQuery.graphql"
 
-export interface RegisterQueryResponse {
-  sale: redirects_sale
-  me: redirects_me
+type Omit<T, K> = Pick<T, Exclude<keyof T, K>>
+type DeFraged<T> = Omit<T, " $refType" | " $fragmentRefs">
+
+export interface DeFragedRegisterQueryResponse {
+  sale: DeFraged<routes_RegisterQueryResponse["sale"] & Register_sale>
```

Comment:
Our effort to remove confusing relay-specific attributes from these objects. If it's overkill, we will remove.